src/cli.rs

```diff
@@ -342,6 +342,12 @@ pub fn cli() -> Command {
         .subcommand(Command::new("list")
                     .alias("ls")
                     .about("Get list of installed packages in a snapshot")
+                    .arg(Arg::new("exclude-dependency")
```

Comment:
Can you add little document/comment here?

Reply:
.help is a comment.